### ISSUE TITLE
How do I customize the loss function in python to achieve the same result as'objective':'binary:logistic'

### ISSUE THREAD

**Issue by qq73927267** [dmlc/xgboost#6436]

**Comment by hcho3** [comment]
See https://github.com/dmlc/xgboost/blob/master/demo/guide-python/custom_objective.py.
